docs/docs/get_started.md

```diff
@@ -68,37 +62,27 @@ To activate the environment in your terminal/powershell:
     Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
     ```
 
-See [venv documentation](https://docs.python.org/3.10/library/venv.html#creating-virtual-environments) for more information.
-
-## Install python dependencies in the environment
+See [venv documentation](https://docs.python.org/3.10/library/venv.html#creating-virtual-environments){:target="_blank"} for more information.
 
 Once the environment is activated, install the project dependency:
 
 ```
 pip install -r requirements.txt
 ```
 
-## ARLAS Stack
-
-The tutorials require an up and running ARLAS Stack. 
-
-### ARLAS Cloud
-
-The simplest way is to create an ARLAS Cloud account.
+## ARLAS Instance
 
-See [ARLAS Cloud Guide](https://docs.arlas.io/static_docs/arlas_cloud/)
+The tutorials require ARLAS up and running.
 
-### Install and run ARLAS Exploration Stack (Optional)
+ARLAS can be managed or deployed locally:
```

Comment:
Remove "ARLAS can be managed or deployed locally:"
change with:
You can either use the ARLAS cloud service managed by Gisaïa or deploy the ARLAS software stack on your infrastructure/computer:
- **ARLAS Cloud**: The simplest way to access an ARLAS instance is to create an ARLAS Cloud account. See [ARLAS Cloud Guide](../../static_docs/arlas_cloud.md)
- **Run ARLAS stack**: To run a simple ARLAS stack and Elasticsearch on your machine, follow the [Deploying ARLAS Guide](../../external_docs/ARLAS-Exploration-stack/arlas_exploration_stack.md).